atc/api/config_test.go

```diff
@@ -384,7 +384,35 @@ var _ = Describe("Config API", func() {
 									}
 								]
 							}`))
+					})
+				})
+				Context("and is an empty string", func() {
+					BeforeEach(func() {
+						var err error
+						request, err = requestGenerator.CreateRequest(atc.SaveConfig, rata.Params{
+							"team_name":     "",
+							"pipeline_name": "",
+						}, nil)
+						Expect(err).NotTo(HaveOccurred())
+
+						request.Header.Set("Content-Type", "application/json")
+
+						payload, err := json.Marshal(pipelineConfig)
+						Expect(err).NotTo(HaveOccurred())
+
+						request.Body = gbytes.BufferWithBytes(payload)
+					})
+
+					It("returns warnings in the response body", func() {
+						Expect(ioutil.ReadAll(response.Body)).To(MatchJSON(`
+							{
+								"errors": [
+										"identifier cannot be an empty string"
```

Comment:
There seems to be a few failing tests now as a result of adding the context to the error: https://ci.concourse-ci.org/builds/69020323#L60011778:17:83

(just requires changing the asserted error message)